fuzz/Cargo.lock

```diff
@@ -122,5 +123,3 @@ checksum = "8ccb82d61f80a663efe1f787a51b16b5a51e3314d6ac365b08639f52387b33f3"
 [[package]]
 name = "virtue"
 version = "0.0.7"
```

Comment:
This doesn't seem to have regenerated the lock file since its still pointing to virtue 0.0.7.